### ISSUE TITLE
DockerToolbox-1.8.2d Windows 10

### ISSUE THREAD

**Issue by lucidprogrammer** [docker/toolbox#222]
Previous version of DockerToolbox was removed, including previous versions of Git and Virtual Box. All left behind folders were also deleted and this is a clean install. Installation succeeded without any issue.

However, while running Kitematic for the first time, I get the error from Virtualbox. "Cannot rename this connection.The requested operation was made in the context of a transaction that is no longer active." See the screenshot attached.

![image](https://cloud.githubusercontent.com/assets/6227632/10331431/2e750d56-6ccd-11e5-91d9-4a44d1c9e8ba.png)

Kitematic waits at 99% and finally comes with an error message as follows. However, from the docker quick start terminal, able to ssh into the default vm.[docker-machine ls shows 
default               virtualbox     Running   tcp://192.168.99.100:2376]

However, this is not usable for the reason that, cannot access it on host. For example if I run 
 docker run -d -p 8080:80 tutum/apache-php, I am not able to access the resource from http://192.168.99.100:8080. This issue was for 1.8.2c. I am going back to DockerToolbox-1.8.1c which was the only version which has worked without much of an issue. Others all had issues like hanging etc. This issue is specific to DockerToolbox-1.8.2d Windows 10. Really appreciate if this can be properly fixed as this trial and error is really eating up the development time.

![image](https://cloud.githubusercontent.com/assets/6227632/10331877/cc1f752a-6cd0-11e5-850e-78acc7d711e8.png)

**Comment by hantsy** [comment]
I got the same problem when run QuickStart scripts, even I clean my VirtualBox reg info via regedit.

**Comment by mchiang0610** [comment]
Merging with #153

**Comment by kundeng** [comment]
Why merging with #153,  this issue is still there and doesn't seem related to that described in #153.

**Comment by lucidprogrammer** [comment]
Yes itis not related to #153

**Comment by pyrotek** [comment]
I'm also stuck here. Running Windows 10. I've tried clearing out the network connections from VBox settings, removing all VM related files from .docker/, and even re-installing Docker with no luck. I also tried examining other windows components that have the concept of transactions, such as Component Services, but no luck there either. Next going to try reinstalling Virtual Box, but I think this has something to do with a stuck system component or service. 

When I view the *running* services in task manager I see a stopped UserDataSvc and UnistoreSvc services as well as started services with the same names, but appended with "..._Session4". Could this be the cause.

**Comment by pyrotek** [comment]
The services I mentioned seem to be native services to Windows, so likely not the culprit. I reinstalled everything including VirtualBox, but got the same error. While the error was showing, I used Process Explorer to determine that the message was originating from VBoxSVC.exe. 

I'll keep poking around, but I guess my next real step is to reinstall Windows. Luckily this is a recent upgrade and there is really not much installed.
